superset/commands/report/execute.py

```diff
@@ -142,14 +142,15 @@ def update_report_schedule_slack_v2(self) -> None:
                     channel_names = (slack_recipients["target"] or "").replace("#", "")
                     # we need to ensure that existing reports can also fetch
                     # ids from private channels
-                    channels = get_channels_with_search(
+                    channels_data = get_channels_with_search(
                         search_string=channel_names,
                         types=[
                             SlackChannelTypes.PRIVATE,
                             SlackChannelTypes.PUBLIC,
                         ],
                         exact_match=True,
                     )
+                    channels = channels_data["result"]
```

Comment:
It looks to me like get_channels_with_search is supposed to return a list? `list[SlackChannelSchema]`